service-templates/load-balanced-fargate-svc/v1/instance_infrastructure/cloudformation.yaml

```diff
@@ -110,6 +110,41 @@ Resources:
               - Effect: 'Allow'
                 Action: 'sns:Publish'
                 Resource: '{{environment.outputs.SNSTopicArn}}'
+      PermissionsBoundary: !Ref TaskRolePermissionBoundaryPolicy
+      ManagedPolicyArns:
+        - !Ref BaseTaskRoleManagedPolicy
+        {{ service_instance.components.custom_runtime.outputs
```

Comment:
I think component outputs are now referenced with `service_instance.components.default.outputs`?

Reply:
oh yes you are correct, good catch, thanks!

Reply:
fixed